spec/index.bs

```diff
@@ -1277,6 +1277,12 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
     <tr class=row-continuation><td colspan=4>
         The maximum value for the arguments of {{GPUComputePassEncoder/dispatch(x, y, z)}}.
 
+    <tr><td><dfn>maxMultiDrawIndirectCount</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>1
```

Comment:
If 1 is permitted, does it mean we can make it a core API?
I think we should make a decision here. Either leave 1 and make it core, or make it an optional feature, lift the limit entirely (see https://vulkan.gpuinfo.org/displaydevicelimit.php?name=maxDrawIndirectCount&platform=all). I.e. it becomes limited in the same way as maximum index count is limited.
I believe that latter path is the way to go.

Reply:
Even with 1 being permitted it cannot be core, at least not without some emulation with a compute shader to read the `drawCountBuffer` and write to the `instanceCount`. This is because if `maxDrawCount` is 1 the value in `drawCountBuffer` can be 0 or 1.

Reply:
Interestingly, (at least of) the implementations would need to write to `instanceCount` from a compute shader regardless of this, since they need to guarantee that there is enough instance and index data for each indirect call. So it's possible that doing this 0/1 work is going to be easy on top of that, but I don't want to piggy back on that mechanism yet, since nobody have implemented it.

Reply:
If doing this, even the `drawCountBuffer` could be pollyfilled by setting `instanceCount` to 0 for all draws after the value in `drawCountBuffer`.